tests/test_core.py

```diff
@@ -1039,7 +1054,8 @@ def test_partial_method_exception(self):
             "Invalid selector overloaded:invalidArgument:. Available selectors are: "
             "overloaded, overloaded:, overloaded:extraArg:, "
             "overloaded:extraArg1:extraArg2:, overloaded:extraArg2:extraArg1:, "
-            "overloaded:orderedArg1:orderedArg2:, overloaded:duplicateArg:duplicateArg:",
+            "overloaded:orderedArg1:orderedArg2:,"
```

Comment:
This missed a space in the collapse into a new line:
```suggestion
            "overloaded:orderedArg1:orderedArg2:, "
```